### DISCUSSION TITLE
Improved Keybindings

### DISCUSSION THREAD

**Discussion by FuckingToasters** [prasanthrangan/hyprdots#669] [Show and tell]
Hi, so i changed the keybindings a bit and i guess they could be useful for other's aswell so i'm going to share them here.

1. Win+E opens Thunar Filemanager, The default Dolphin looks a bit weird.
2. Win+C opens Codium instead of code, which esenteally is the same but opensource and without the Microsoft
3. Win+D Opens your discord client of choice, personally, i use armcord which is a fork of vencord
4. Win+T opens Telegram Desktop
5. Win+Return open your Terminal of Choice

Other Keybindings are the defaults, i may change more stuff in the future but for now I'm satisfied with my setup.
[keybindings.conf.txt](https://github.com/prasanthrangan/hyprdots/files/13704209/keybindings.conf.txt)

Note: Github don't support uploading .conf files so a added .txt, to use it, you need to remove the txt filetype.

**Comment by kRHYME7** [comment]
Nice! You can use this also https://gist.github.com/ to upload some configs. Thanks!

**Comment by akashgagda** [comment]
Installations script work with codium like theme extension install or theme switch ?

**Comment by mahaveergurjar** [comment]
No, I think these keybindings are good and need to change the user can change in userperfs file or keybindings.conf file
Like this type 
![231225_10h12m55s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/44203d4f-0596-4e06-ba31-9d3064973744)
![Uploading 231225_10h13m36s_screenshot.png…]()

**Comment by mahaveergurjar** [comment]
I think only need to change the keybindings of vscode and clipboard
beacuse super key is used for applications super + alt is used for scripts super + shift is used for theme and customisation so I think super+v is vs code and super + alt is for clipboard

**Comment by 4thehalibit** [comment]
Codeium is missing many extensions. I suggest keeping it and turning off telemetry if that is your concern

**Comment by FuckingToasters** [comment]
> Codeium is missing many extensions. I suggest keeping it and turning off telemetry if that is your concern

You can always download the extension .vsix file from Microsoft Marketplace and import it in vscodium.
